1208-projects/user_opens_my_projects.feature

```diff
@@ -0,0 +1,73 @@
+Feature: User opens My Projects
+
+    @xxx
+    Scenario: User opens My Projects
+        Given that a User is logged in
+        When they navigate to the Projects Page
+        Then they see a Create New Project Button
+        And they see a search field
+        And they see a list of Active Projects
+        And they see a collapsed list of Concluded Projects
+        And the lists contains Projects where the User has the role as:
+            | Project Manager       |
+            | Local Project Manager |
+            | Participant           |
+        And each Project has:
+            | Title           |
+            | Institution     |
+            | Project Manager |
+        And the list can be sorted by Title, Institution and Project Manager
+        And they see a Next, Previous and needed increments to support pagination of the items in the list
+
+    @xxx
+    Scenario: User execute a search 
+        Given User opens My Projects
+        When the user execute a Search
+        Then the Search term is displayed as a Filter
+        And the user may delete the Filter
+        And the user may delete all active Filters
+        And the list of Active and Concluded Projects is replaced by a result list of Projects matching the search text
+        And the lists contains Projects where the User has the role as:
+            | Project Manager       |
+            | Local Project Manager |
+            | Participant           |
+        And each Project has:
+            | Title           |
+            | Institution     |
+            | Project Manager |
+        And the list can be sorted by Title, Institution and Project Manager
+        And they see a Next, Previous and needed increments to support pagination of the items in the list
```

Comment:
Det meste her er dekket av scenario over. foreslår å snevre inn denne til å kun handle om selve filtreringen. støtte for flere filter er kanskje ikke nødvendig? tror man kommer langt ved å filterere med fritekst

```suggestion
    Scenario: User searches for Projects
        Given User opens My Projects
        When they enter a search term in the search field
        Then they only see Projects where the search term is a substring of the Project's:
            | Title           |
            | Project Manager |
```

Reply:
vet ikke hvilke felter som er relevant å filtrere mot

Reply:
Justert i ny commit